alto/commands/cromwell/list_jobs.py

```diff
@@ -0,0 +1,164 @@
+        else:
+            query_data.append({"additionalQueryResultFields": "labels"})
+            query_data.append({"label": f"creator:{username}"})
+
+    if job_status is not None:
+        query_data.append({"status": job_status})
+    resp = requests.post(
+        f"http://{server}:{port}/api/workflows/v1/query",
+        json=query_data,
+    )
+    resp_dict = resp.json()
+    for res in resp_dict["results"]:
+        if "labels" in res:
+            res["creator"] = res["labels"].get("creator", "")
+            del res["labels"]
+        else:
+            res["creator"] = ""
+        res["submission"] = datetime_from_utc_to_local(res.get("submission", ""))
+        res["start"] = datetime_from_utc_to_local(res.get("start", ""))
+        res["end"] = datetime_from_utc_to_local(res.get("end", ""))
+    if resp.status_code == 200:
+        df_jobs = pd.DataFrame.from_records(resp_dict["results"])
+        show_jobs(df_jobs)
+    else:
+        print(resp_dict["message"])
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description="List jobs submitted to the server.")
+    parser.add_argument(
+        "-s",
+        "--server",
+        dest="server",
+        action="store",
+        required=True,
+        help="Server hostname or IP address.",
+    )
+    parser.add_argument(
+        "-p",
+        "--port",
+        dest="port",
+        action="store",
+        default="8000",
+        help="Port number for Cromwell service. The default port is 8000.",
+    )
+    parser.add_argument(
+        "-a",
+        "--all",
+        dest="is_all",
+        action="store_true",
+        default=False,
+        help="List all the jobs on the server.",
+    )
+    parser.add_argument(
+        "-u",
+        "--user",
+        dest="user",
+        action="store",
+        default=getpass.getuser(),
+        help="List jobs submitted by this user.",
+    )
+    parser.add_argument(
+        "--id",
+        dest="job_id",
+        action="store",
+        help="List all the subtasks of the specified job.",
+    )
+    parser.add_argument(
+        "--only-succeeded",
+        dest="only_succeeded",
+        action="store_true",
+        default=False,
+        help="Only show jobs succeeded.",
+    )
+    parser.add_argument(
+        "--only-running",
+        dest="only_running",
+        action="store_true",
+        default=False,
+        help="Only show jobs that are running.",
+    )
+    parser.add_argument(
+        "--only-failed",
+        dest="only_failed",
+        action="store_true",
+        default=False,
+        help="Only show jobs that have failed.",
+    )
+
+    args = parser.parse_args(argv)
+
+    job_status = None
+    if args.only_succeeded:
+        job_status = "Succeeded"
+    elif args.only_failed:
+        job_status = "Failed"
+    elif args.only_running:
+        job_status = "Running"
+
+    list_jobs(args.server, args.port, args.is_all, args.user, args.job_id, job_status)
```

Comment:
Similarly, since we don't need `--id` option, please remove `args.job_id` and the corresponding argument in `list_jobs` definition.